compiler/hie-core/test/exe/Main.hs

```diff
@@ -6,12 +6,15 @@
 module Main (main) where
 
 import Control.Monad (void)
+import Control.Monad.IO.Class (liftIO)
 import qualified Data.Text as T
+import Control.Lens
```

Comment:
Avoiding the dependency on Lens is probably a good idea. You seem to mostly be using it to do selector functions anyway, so it should be just as simple without,

Reply:
Done.